g11n-ws/vip-manager-l10n-agent/src/main/java/com/vmware/l10agent/schedule/SourceRequestCron.java

```diff
@@ -37,7 +44,8 @@
 @Service
 public class SourceRequestCron {
 	private static Logger logger = LoggerFactory.getLogger(SourceRequestCron.class);
-
+    private final static String instructV1 = "v1";
+    private final static String instructS3 = "s3";
```

Comment:
Is it possible to define these constants in the PropertyContantKeys file？

Reply:
It only use in this class